Framework/API/test/AsynchronousTest.h

```diff
@@ -11,12 +11,21 @@
 #include <Poco/NObserver.h>
 #include <Poco/Thread.h>
 
+#include <condition_variable>
+#include <mutex>
+
 using namespace Mantid::Kernel;
 using namespace Mantid::API;
 using namespace std;
```

Comment:
Oh yes, of course. :man_facepalming:

That's okay. At least they are taken out of the global scope and won't clash with other variables in other test headers (The final test binary includes all of the headers in a single file).